src/ruby.ts

```diff
@@ -36,24 +36,29 @@ export class Ruby {
           await this.activate("rvm-auto-ruby");
           break;
         default:
-          await this.activateShadowenv();
+          await this.activateFallback();
           break;
       }
 
-      await this.delay(500);
-      await this.displayRubyVersion();
-      this.checkYjit();
+      vscode.window.setStatusBarMessage(`Ruby ${this.rubyVersion}`);
     } catch (error: any) {
       vscode.window.showErrorMessage(
         `Failed to activate ${this.versionManager} environment: ${error.message}`
       );
     }
   }
 
-  private async activateShadowenv() {
-    await vscode.extensions
-      .getExtension("shopify.vscode-shadowenv")
-      ?.activate();
+  private async activateFallback() {
```

Comment:
Let's continue to call it `activateShadowenv`.

Reply:
Another option is to add another branch for shadowenv and have the default do nothing. In that case, we won't be using a version manager and will just pick up whatever is currently set in the VS Code environment.